recipes/recipes_emscripten/lz4-c/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -ex
+
+export CFLAGS="${CFLAGS} -O3 -fPIC"
+export LDFLAGS="${LDFLAGS} -lrt"
+
+# Build
+make -j${CPU_COUNT} PREFIX=${PREFIX} VERBOSE=1
```

Comment:
Not sure if it should be:
```suggestion
VERBOSE=1 make -j${CPU_COUNT} PREFIX=${PREFIX}
```

Reply:
The way you have it seems to be working already